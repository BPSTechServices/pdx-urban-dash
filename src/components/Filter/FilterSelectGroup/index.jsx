import React from "react";
import PropTypes from "prop-types";
import { Col } from "reactstrap";

export default class FilterSelectGroup extends React.Component {
  static propTypes = {
    title: PropTypes.string.isRequired,
    hidden: PropTypes.bool
  };

  constructor(props) {
    super(props);

    this.title = "";
    this.hidden = false;
  }

  render() {
    if (this.props.hidden) return null;

    return (
      <Col style={{ marginLeft: "1em" }}>
        <h4 style={{ marginBottom: "1rem" }}>{this.props.title}</h4>
        <hr />
        {this.props.children}
      </Col>
    );
  }
}