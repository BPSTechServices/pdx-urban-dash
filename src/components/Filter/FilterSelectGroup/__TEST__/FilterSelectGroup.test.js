import React from 'react';
import { shallow } from 'enzyme';
import FilterSelectGroup from '../index';

const testFunc = () => { console.log(); };

describe('FilterSelectGroup Render', () => {
  test('snapshot', () => {
    const wrapper = shallow(<FilterSelectGroup title={'test'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
