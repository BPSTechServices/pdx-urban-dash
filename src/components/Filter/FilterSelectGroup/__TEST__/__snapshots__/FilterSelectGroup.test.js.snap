// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterSelectGroup Render snapshot 1`] = `ShallowWrapper {}`;
